rules/salary-sacrifice-electronic-devices/rule.md

```diff
@@ -34,13 +32,8 @@ In Australia, salary sacrificing is when your employer agrees to pay for your de
 Most companies will let you salary sacrifice FBT-exempted portable electronic devices that:
 
 * Are used predominantly for work-related purposes
-
 * Are easily portable and designed for use away from an office environment
-
-* Are small and light
```

Comment:
> ATO doesn't explicitly say that salary sacrificed devices need to be thin and light.

Yes, it does. see 
https://www.ato.gov.au/businesses-and-organisations/hiring-and-paying-your-workers/fringe-benefits-tax/exemptions-concessions-and-other-ways-to-reduce-fbt/work-related-items-exempt-from-fbt

There should be a link to this page here

Reply:
Oh, my mistake. I found it.